back-end/handlers/playerHelpers.ts

```diff
@@ -1,19 +1,46 @@
 import playerDb from '../db/player.ts';
 import hostDb from '../db/host.ts';
 import IGame from '../interfaces/IGame.ts';
+import Player from '../models/Player.ts';
 import { PlayerStates } from '../interfaces/IPlayerState.ts';
 import { Server } from 'socket.io';
 
 export default {
   allPlayersGoToNextQuestion: async (gameId: number, io: Server): Promise<void> => {
+
     const currentGameData: IGame | null = await hostDb.getGameData(gameId);
     if (currentGameData === null) {
       return;
     }
 
     const currentQuestionIndex = currentGameData.currentQuestionIndex;
     const quizQuestionOptionsText: string[] = currentGameData.quizQuestions[currentQuestionIndex].optionsList || [];
+    const allPlayersInGame = await playerDb.getPlayers(gameId);
+    for (let i = 0; i < allPlayersInGame.length; i++) {
+      const player = allPlayersInGame[i]
 
-    await playerDb.updateAllPlayerStates(gameId, PlayerStates.SeeingQuestion, io, { quizQuestionOptionsText });
+      if (allPlayersInGame[i].id === currentGameData.quizQuestions[currentQuestionIndex].playerId) {
+        await Player.updateOne({
```

Comment:
You could refactor this so you only have to call `Player.updateOne` in one place, with a variable for the state to set.